src/AIS_Decoder.cpp

```diff
@@ -2238,14 +2237,11 @@ void AIS_Decoder::OnTimerAIS( wxTimerEvent& event )
                     current_targets->erase( it );
```

Comment:
Yes, you are correct. That is why the original author reset the iterator to the beginning. I will fix and resubmit.